pkg/auth/providers/saml/saml_actions.go

```diff
@@ -46,6 +48,18 @@ func (s *Provider) testAndEnable(actionName string, action *types.Action, reques
 		return err
 	}
 
+	if s.hasLdapGroupSearch() {
+		ldapConfig, certpool, err := ldap.GetLDAPConfig(s.ldapProvider)
+		if err != nil {
+			return err
+		}
```

Comment:
`hasLdapGroupSearch` only checks the provider name and not the presence of ldapconfig fields, so this will return an error even if user didn't want to configure openldap support?

Reply:
youre right, should just Debug log here and in the call under it just like we do in the Configure